tinygrad/ops.py

```diff
@@ -73,6 +73,7 @@ def stride(self, _): raise NotImplementedError
 
 class _Device:
   def __init__(self) -> None: self._buffers: List[str] = [x.stem[len("ops_"):].upper() for x in (pathlib.Path(__file__).parent/"runtime").iterdir() if x.stem.startswith("ops_")]
+  @functools.lru_cache(maxsize=None)  # this class is a singleton, pylint: disable=method-cache-max-size-none
```

Comment:
So this actually breaks things. What if I change `self.DEFAULT`?

Can you add a test to make sure changing `self.DEFAULT` keeps working?

Reply:
`cached_property` is subtly different from the normal property decorator as it allows you to write to the property, so its possible to change Device.DEFAULT externally